### ISSUE TITLE
can't print A4 landscape via CUPS

### ISSUE THREAD

**Issue by harriott** [nomacs/nomacs#1179]
## the bug
printing A4 landscape prints to cropped A4 portrait

## To Reproduce
1. get [Distribution_of_the_global_biomass.png](https://en.wikipedia.org/wiki/Biomass_%28ecology%29#/media/File:Distribution_of_the_global_biomass.png) and prepare to print (`ctrl+p`)
    1. `Page Setup` correctly reports `Orientation > Landscape`
    1. `Print > Properties` correctly reports `Orientation > Landscape`
    1. print gets done `Portait`, the image correctly sized, but with sides of the image necessarily cropped off
1. using [CUPS](https://en.wikipedia.org/wiki/CUPS) directly ([lp(1)](https://www.cups.org/doc/man-lp.html)): `lp Distribution_of_the_global_biomass.png` prints correctly as landscape

## Expected behavior
`nomacs` should correctly print the image `Landscape`, as was the case in previous years

## Screenshots
![Page_Setup](https://github.com/user-attachments/assets/ba951cd5-a12e-4067-b622-71b5dd6c3ab2)
![Printer_Properties](https://github.com/user-attachments/assets/96a62393-71bc-4823-8cdb-68e3bb0b6b5c)

## Desktop
- `Arch Linux v6.11.3`
- `Openbox v3.6.1`
- `nomacs v3.19.1`

**Comment by scrubbbbs** [comment]
Current version works for me printed to PDF file, Ubuntu 24.04, Qt 6.9.1 CUPS v2.4.7

**Comment by harriott** [comment]
## working now
- `linux 6.16.4.arch1-1`
- `nomacs v3.21.0`
- `Xfce v4.20` - I'm no longer using `Openbox`

That landscape image prints correctly from `nomacs` - no idea why.
But there's now a minor snag in the text positioning of the settings windows:

<img width="440" height="502" alt="Image" src="https://github.com/user-attachments/assets/5d09d1fb-4071-4899-a44f-3897fb59331c" />

<img width="524" height="579" alt="Image" src="https://github.com/user-attachments/assets/aa83ca0a-1183-4632-8e11-8fefd8638763" />

**Comment by scrubbbbs** [comment]
Thanks that is a known issue with the Dark/Light theme (#1243). The system theme works correctly.

**Comment by harriott** [comment]
Yes, `Edit > Settings > GENERAL > Nomacs Theme > System` fixed those text positionings, thanks.
